packages/handbrake.rb

```diff
@@ -3,55 +3,102 @@
+  depends_on 'intel_media_sdk' if ARCH == 'x86_64'
+  depends_on 'jansson' # R
+  depends_on 'libass' # R
+  depends_on 'libdvdcss'
+  depends_on 'libgudev' # R
+  depends_on 'libjpeg' # R
+  depends_on 'libmp3lame' # R
+  depends_on 'libogg' # R
+  depends_on 'libpng' # R
+  depends_on 'libtheora' # R
+  depends_on 'libva' # R
+  depends_on 'libvorbis' # R
+  depends_on 'libvpx' # R
+  depends_on 'libx264' # R
+  depends_on 'libxml2' # R
   depends_on 'mesa'
-  depends_on 'xcb_util'
-  depends_on 'freetype'
+  depends_on 'nasm' => :build
+  depends_on 'numactl' # R
+  depends_on 'onevpl' if ARCH == 'x86_64' # R
+  depends_on 'opus' # R
+  depends_on 'pango' # R
+  depends_on 'speex' # R
+  depends_on 'util_linux' # R
+  depends_on 'wayland_protocols' => :build
+  depends_on 'xcb_util' => :build
+  depends_on 'xzutils' # R
+  depends_on 'zlibpkg' # R
 
-  def self.patch
-    system "for f in \$(find -name '*.*'); do sed -i 's,/usr/include/libxml2,#{CREW_PREFIX}/include/libxml2,g' \$f; done"
-  end
+  no_env_options
 
   def self.build
-    system "env CFLAGS='-pipe -flto=auto' CXXFLAGS='-pipe -flto=auto' \
-      ./configure --prefix=#{CREW_PREFIX} --enable-x265 --enable-numa --enable-fdk-aac --harden"
-    Dir.chdir 'build' do
-      system 'make'
+    system "#{CREW_ENV_FNO_LTO_OPTIONS} ./configure #{CREW_OPTIONS} \
+      --enable-x265 \
+      --enable-numa \
+      --enable-fdk-aac \
+      --enable-qsv \
+      --no-harden \
+      --force"
+    FileUtils.mkdir_p 'x86_64-cros-linux-gnu/contrib/lib/pkgconfig'
+    Dir.chdir('x86_64-cros-linux-gnu/contrib/lib/pkgconfig') do
+      @handbrake_libs = %w[glib-2.0 fribidi harfbuzz freetype]
+      @handbrake_libs.each do |f|
+        next if File.file?("#{f}.pc")
+
+        FileUtils.ln_sf "#{CREW_LIB_PREFIX}/pkgconfig/#{f}.pc", "#{f}.pc"
+      end
     end
+    system 'make -C x86_64-cros-linux-gnu || make -j1 -C x86_64-cros-linux-gnu'
   end
 
   def self.install
-    Dir.chdir 'build' do
+    Dir.chdir 'x86_64-cros-linux-gnu' do
       system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     end
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d"
+    File.write "#{CREW_DEST_PREFIX}/etc/env.d/10-handbrake", <<~'HANDBRAKE_ENVD_EOF'
+      alias ghb="GDK_BACKEND=wayland ghb"
+    HANDBRAKE_ENVD_EOF
   end
+end
 
-  def self.postinstall
-    puts
-    puts "To get started, type 'ghb'.".lightblue
-    puts
-    puts "Type 'HandBrakeCLI' for the command line.".lightblue
-    puts
-    puts 'To complete the installation, execute the following:'.lightblue
-    puts "echo 'alias ghb=\"WAYLAND_DISPLAY=wayland-0 DISPLAY=\'\' GDK_BACKEND=wayland ghb\"' >> ~/.bashrc".lightblue
-    puts
-  end
+def self.postinstall
+  puts
+  puts "To get started, type 'ghb'.".lightblue
+  puts
+  puts "Type 'HandBrakeCLI' for the command line.".lightblue
+  puts
+  puts 'Please run the following to finish the install:'.orange
+  puts "source #{CREW_PREFIX}/etc/env.d/10-handbrake".lightblue
+  puts
```

Comment:
What happened to the indentation?  This should be inside the class block, correct?